pos_attachment/static/src/app/screens/product_screen/attachments_button.esm.js

```diff
@@ -0,0 +1,42 @@
+/** @odoo-module */
+/*
+    Copyright 2024 Dixmit
+    License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+*/
+import {AttachmentPopup} from "../../utils/attachment_popup/attachment_popup.esm";
+import {Component} from "@odoo/owl";
+import {ConnectionLostError} from "@web/core/network/rpc_service";
+import {ProductScreen} from "@point_of_sale/app/screens/product_screen/product_screen";
+import {usePos} from "@point_of_sale/app/store/pos_hook";
+import {useService} from "@web/core/utils/hooks";
+
+export class AttachmentsButton extends Component {
+    setup() {
+        this.pos = usePos();
+        this.popup = useService("popup");
+    }
+
+    get attachments() {
```

Comment:
naming : if i understand correctly, this is not attachments, but attachment_qty. Don't you think ?